studio/components/interfaces/Organization/TeamSettings/InviteMemberButton.tsx

```diff
@@ -24,10 +24,13 @@ const InviteMemberButton: FC<Props> = ({ user, members = [], roles = [], rolesAd
   const { slug } = router.query
 
   const enablePermissions = useFlag('enablePermissions')
+  const isOwner = ui.selectedOrganization?.is_owner
 
   const [isOpen, setIsOpen] = useState(false)
   const { mutateOrgMembers } = useOrganizationDetail((slug as string) || '')
-  const canInviteMembers = checkPermissions(PermissionAction.CREATE, 'user_invites')
+
+  // [Joshen] The check against user_invites and auth.subject_roles are the same
+  const canAddMembers = enablePermissions ? rolesAddable.length > 0 : isOwner
```

Comment:
This hardcodes using the role instead of checking the permission. We could check whether there's a permission that allows inviting as some role:

`roles.some(({ id: role_id }) => checkPermissions(Permission.CREATE, 'user_invites', { resource: { role_id } })`